### ISSUE TITLE
Canvas: Grouping element

### ISSUE THREAD

**Issue by pgnam258** [grafana/grafana#90218]
**Why is this needed**: Easy to move a group of elements to adjust

[Add a description of the problem the feature is trying to solve.]

**What would you like to be added**: A button/toggle to enable/disable this feature

[Add a brief description of what the feature or enhancement should do.]

**Who is this feature for?**: A SystemAdmin like me, I want to show my manager a picture of full view about our infrastructure system using Canvas

[Add information on what kind of user the feature is for.]

**Comment by jeremow** [comment]
I agree for the feature. In Layers: elements, should be nice to make groups of different elements. Or at least, we could select multiple elements with Control key
